### PR TITLE
Record the resource template responses into the Redux state

### PR DESCRIPTION
**Pull request by jcoyne**
This will allow us to validate the redux state without having to reload resouce templates.

### CODE REVIEW

File: __tests__/GraphBuilder.test.js [hunk 1]
```diff
@@ -7,49 +7,60 @@ const rdf = require('rdf-ext')
...
-              uri: 'http://id.loc.gov/vocabulary/millus/gnt',
-            },
-          ],
+      entities: {
+        resourceTemplates: {
+          'resourceTemplate:bf2:Monograph:Work': {
+            resourceURI: 'http://id.loc.gov/ontologies/bibframe/Work',
+          },
+          'resourceTemplate:bf2:Note': {
+            resourceURI: 'http://id.loc.gov/ontologies/bibframe/Note',
```
Comment (justinlittman) [inline __tests__/GraphBuilder.test.js]: So just to be clear: This is just a stand-in for the entire resource template?

File: __tests__/reducers/index.test.js [hunk 2]
```diff
@@ -112,22 +137,36 @@ describe('selectorReducer', () => {
...
+        },
       },
     )
   })
 
   it('handles SET_BASE_URL', () => {
     inputs.setBaseURL = jest.fn().mockReturnValue({})
     const oldState = {
-      'resourceTemplate:bf2:Monograph:Instance': {
-        rdfClass: 'http://id.loc.gov/ontologies/bibframe/Instance',
```
Comment (justinlittman) [inline __tests__/reducers/index.test.js]: Goodbye rdfClass!

File: __tests__/reducers/index.test.js [hunk 3]
```diff
@@ -336,28 +385,32 @@ describe('populatePropertyDefaults()', () => {
...
-
-      expect(newState).toEqual({
-        'resourceTemplate:bf2:Monograph:Work': {
-          'http://id.loc.gov/ontologies/bibframe/title': {},
-          rdfClass: 'http://id.loc.gov/ontologies/bibframe/Work',
-        },
-      })
-    })
-  })
+  /*
```
Comment (justinlittman) [inline __tests__/reducers/index.test.js]: Cruft?

File: src/reducers/index.js [hunk 4]
```diff
@@ -90,31 +89,39 @@ export const refreshResourceTemplate = (state, action) => {
...
   const newState = { ...state }
-  const reduxPath = action.payload.reduxPath
 
-  const lastObject = reduxPath.reduce((newState, key) => newState[key] = newState[key] || {}, newState)
+  // Call payload just to avoid an eslint warning
+  action.payload
 
-  lastObject.rdfClass = action.payload.resourceURI
+  /*
+   * TODO: fix this
```
Comment (justinlittman) [inline src/reducers/index.js]: Just a reminder to fix this or ticket separately.

File: __tests__/GraphBuilder.test.js (hunk 1 above)
Comment (jcoyne) [reply __tests__/GraphBuilder.test.js]: right

File: src/reducers/index.js (hunk 4 above)
Comment (jcoyne) [reply src/reducers/index.js]: This was the part I was hoping you would fix for me 😆

Comment (justinlittman) [reply src/reducers/index.js]: Can't we just get rid of setResourceURI altogether?

**Comment by jcoyne** [comment]
@justinlittman Changes have been made.

**Pull request merged**